contracts/BalanceTrackerBase.sol

```diff
@@ -55,7 +54,7 @@ abstract contract BalanceTrackerBase {
     // Collected fees
     uint256 public collectedFees;
     // Reentrancy lock
-    uint256 internal _locked = 1;
```

Comment:
Transient changes state during a transaction only, and then rolls back to the previous value. It also consumes only 100 gas to change the value.